### DISCUSSION TITLE
D.S. is unavailable currently after updated

### DISCUSSION THREAD

**Discussion by qiao-liang233** [ppy/osu#27342] [Q&A] [answered]
i don't know if something is wrong,but when mapping i can't change the D.S. quickly or know the D.S. easily.btw the new icon is bad:(

**Comment by Gabixel** [comment]
If by the "icon" you mean the new osu! logo, I'm sorry to hear that. Logo feedback is not really up to debate at the moment (here at least), so please try sticking with it a bit and see if you get used to it and start to like it :)

For the other thing, I don't quite know what is meant here by DS. But searching a little bit, I think what you mean is _Distance Snap/Spacing?_
I do not understand in what way it's not available - looking at the editor, I can see it on the right side in the "compose" screen:

![image](https://github.com/ppy/osu/assets/43073074/d4c8ed46-1f75-4c5a-b931-12da873bdde3)

And you can enable it on the left side:

![image](https://github.com/ppy/osu/assets/43073074/ddda5e32-7e2e-4f95-a601-75820e319fae)

**Reply by qiao-liang233** [reply]
![2024-02-23 (1)](https://github.com/ppy/osu/assets/58883921/769c891b-60bf-4cb3-ab3f-624922722519)
i mean,the button always cant be pressed,i guess its beacause "unavailable"(before it shows the number like "2.42x").i still can change it by the scroll,but not conveniently.
i still need time to accutosm the new logo,hope im wrong

**Reply by cihe13375** [reply]
It could happen if the current DS is 0x or very close to 0x. In the image above, it means the tail of 2 overlaps with the head of 3. Is this the case? Note that the "current DS" is based on the current time (01:21:426), not the note selected (tbh I'm not a fun of this behavior but alright).

For the logo, if you don't care about score submissions etc, I guess the best thing to do is to manually compile lazer and replace the relevant sprites with original ones (I have not tried this yet, but maybe one day)

**Reply by qiao-liang233** [reply]
i do know the case you mentioned.but actually after the update,all the maps i editted before are all in bad situation and in all time the feature is in bad work.
![2024-02-23 (3)](https://github.com/ppy/osu/assets/58883921/a1f35caf-ef62-4a3e-bf63-3799dd7babe1)
[Mili - From a Place of Love (qiao_liang).zip](https://github.com/ppy/osu/files/14387001/Mili.-.From.a.Place.of.Love.qiao_liang.zip)
at the timepoint i am sure the DS is much more than 0x but the button now is still unavailable, so the feature may seem a bug now

**Reply by cihe13375** [reply]
Sorry but I cannot reproduce with the map you provided. I have no idea.

**Reply by bdach** [reply]
If the button is disabled it generally means one of two things:

- The spacing is already correct, and so pressing the button would do nothing anyway
- Adjusting the spacing to match the objects would result in a distance spacing multiplier lower than 0.1x which is not allowed.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/27386
